src/routes/demos/[uuid].ts

```diff
@@ -62,22 +63,15 @@ export const GET = async (requestEvent: RequestEvent) => {
         })
         .catch({
           status: 200,
-          body: { backgroundBrightness: 55, units: 'imperial', cityId: 4887398 }
+          body: { backgroundBrightness: 55, units: 'imperial', cityId: 4887398, SIPQueues: [DEFAULT_SIP_CONFIG] }
```

Comment:
For consistency, `sipQueues`.

Reply:
actually its SIPQueues everywhere.

Reply:
Yeah, I meant consistency with style, i.e. naming variables - pascalCase.